server/conference-data/test.js

```diff
@@ -1,5 +1,7 @@
+/* eslint-disable max-len */
```

Comment:
How would you fix this? The reason for this are https://github.com/redbadger/react.london/pull/197/files#diff-5891332873dc956da5def1a18c9d33d6R136 , https://github.com/redbadger/react.london/pull/197/files#diff-5891332873dc956da5def1a18c9d33d6R103